script/testing/util/db_server.py

```diff
@@ -123,6 +123,7 @@ def stop_db(self, is_dry_run=False):
         return_code = self.db_process.poll()
         if return_code is None:
             self.db_process.terminate()
+            self.db_process.wait()
```

Comment:
I'm wondering if its possible that wait could hang forever with this code. It looks like the default timeout on wait is `None` Should we do something like:
```
try:
    self.db_process.terminate()
    self.db_process.wait(60) #60 second timeout for terminating that throws an error if it times out
except:
    self.db_process.kill()
    self.db_process.wait(60) # If this one times out there are problems and we don't catch the error.
```

Reply:
Yeah, it would be possible for wait to hang. hmm. Sounds like a plan, incorporating.

Reply:
added, thanks!